bigframes/core/compile/compiled.py

```diff
@@ -50,6 +52,7 @@
 T = typing.TypeVar("T", bound="BaseIbisIR")
 
 op_compiler = op_compilers.scalar_op_compiler
+ibis_backend = ibis_bigquery.Backend()
```

Comment:
I'm a bit curious what this is for. IIRC this might cause some authentication to happen, which would break unit tests, but maybe that's been fixed in Ibis?

Reply:
Reverting this aspect of the PR, though didn't notice any issues with authentication.